File: src/datastructure/stack/StackDemo.java
package datastructure.stack;

import java.util.Stack;

public class StackDemo {
    public static void main(String[] args) {
        Stack<Integer> stack = new Stack<>();
        stack.add(-1);
        stack.add(-2);
        stack.push(5);
        stack.add(4);
        stack.push(3);
//        stack.push(2);
//        stack.push(1);
//        stack.push(0);
        System.out.println("Before : "+stack);
        stack.pop();
        System.out.println("After : "+stack);
    }
}

File: src/datastructure/stack/StackDemo.kt
package datastructure.stack

import java.util.*

fun main(){
    val stack = Stack<Int>()
    stack.push(5)
    stack.push(4)
    stack.push(3)
    stack.push(2)
    stack.push(1)
    stack.push(0)

    println(stack.search(1))

    println("Before : $stack")
    stack.pop()
    println("After : $stack")
}